trajectorytools/plot/polar.py

```diff
@@ -0,0 +1,176 @@
+import logging
+
+import numpy as np
+import scipy
+
+from collections import namedtuple
+
+__all__ = ["binned_statistic_polar", "polar_histogram", "plot_polar_histogram"]
+
+
+def remove_nans_from_args(f_unwrapped):
```

Comment:
If you are using np.isfinte you are not only removing NaNs, maybe the name of the function should be more general. 
`remove_nofinite_from_args`?